### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca runcom@linux.com

### CODE REVIEW

**Comment by duglin** [comment]
LGTM if janky is happy

File: registry/registry.go [hunk 1]
```diff
@@ -200,6 +200,7 @@ func DockerHeaders(metaHeaders http.Header) []transport.RequestModifier {
 	return modifiers
 }
 
+// debugTransport is a utility function actually used only in _test
```
Comment (cpuguy83) [inline registry/registry.go]: let's move it to _test then? (assuming this means registry_test.go)

Comment (runcom) [inline registry/registry.go]: yes, wasn't sure to move it

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**